Assets/Scripts/Player/PlayerInventory.cs

```diff
@@ -72,7 +72,8 @@ public class PlayerInventory : MonoBehaviour {
 
     private void InstantiateItem (EquippableItem equippableItem) {
         Transform itemTransform = GetSlotTransform (equippableItem.GetSlot ());
-        Instantiate (equippableItem, itemTransform.position, itemTransform.rotation, this.transform);
```

Comment:
This is to allow the original local rotation of an item to be respected when it is populated into a character slot.